DigitalLearningSolutions.Data/Enums/TrackerEndpointResponse.cs

```diff
@@ -11,8 +11,11 @@ public class TrackerEndpointResponse : Enumeration
         public static readonly TrackerEndpointResponse InvalidAction =
             new TrackerEndpointResponse(-2, nameof(InvalidAction));
 
-        public static readonly TrackerEndpointResponse NullTutorialStatusOrTime =
-            new TrackerEndpointResponse(-14, nameof(NullTutorialStatusOrTime));
+        public static readonly TrackerEndpointResponse StoreAspAssessException =
+            new TrackerEndpointResponse(-6, nameof(StoreAspAssessException));

```

Comment:
What's the deal with these negative ids?

Reply:
When the process is complete, we expect the tracker endpoint to return some text in the format of 
`"[#Result:1]"`
If the process errors, we expect to return a similar output along the lines of
`"[#Result:{errorCode}]"` 
where the `errorCode` is a specific number. These error codes are all negative to indicate the a problem occurred. Hence we save the IDs of each of these as negative so we can easily return the right code from the enum.
Why the numbers are what they are, I am unsure. But I believe these tracker endpoints are all used in Learning Hub, which will expect specific results in specific circumstances - I am unsure how they will behave if any of this changed. But what we have here more or less emulates the old code.